src/lib/Components/Artist/Header.tsx

```diff
@@ -141,6 +126,32 @@ class Header extends React.Component<HeaderProps, State> {
 
     return leadingSubstring + " " + birthday
   }
+
+  @track({ action: "press follow/unfollow button" })
+  handleFollowChange() {
+    const newFollowersCount = this.state.following ? this.state.followersCount - 1 : this.state.followersCount + 1
+    ARTemporaryAPIModule.setFollowArtistStatus(!this.state.following, this.props.artist._id, (error, following) => {
+      if (error) {
+        console.warn(error)
+      } else {
+        this.successfulFollowChange()
+      }
+      this.setState({ following, followersCount: newFollowersCount })
+    })
+    this.setState({ following: !this.state.following, followersCount: newFollowersCount })
+  }
+
+  // currently you can't get state yet, but leaving this in as desired usage
```

Comment:
Does this mean the `state` param in `@track((props, state)` doesn't get passed?

Reply:
Yes and no ^_^. The official and current version of RT doesn't support this no. I've changed this in a local version (which will be submitted back to RT) and it's working 👌.  For the time being I will point this project to that, at which point I will remove the comment. Good catch!

Reply:
Oh cool! Congrats on improving RT at-large 🍾!